open-company/open-company-web
# NUX tweaks

Card: https://trello.com/c/aEGmMimj

To test:
- onboard a new company
- check all the tooltips (landing page, cmail and after first post)
- invite a user
- add Slack team
- add Slack bot
